api/matlab/arrus/BModeDisplay.m

```diff
@@ -3,38 +3,124 @@
     %
```

Comment:
help -> getCineLoop()

Reply:
Comment added:
    % BModeDisplay class includes a cineloop buffer. The content of the \
    % buffer can be accessed through the getCineLoop method.